tests/a_scripts/data_prep/test_convert_delta_to_json.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2022 MosaicML LLM Foundry authors
+# SPDX-License-Identifier: Apache-2.0
+
+# copyright 2022 mosaicml llm foundry authors
+# spdx-license-identifier: apache-2.0
+
+import os
+import sys
+from typing import Any
+
+import unittest
+from unittest.mock import MagicMock, patch
+
+from scripts.data_prep.convert_delta_to_json import stream_delta_to_json
+
+
+class TestStreamDeltaToJson():
+
+    @patch('scripts.data_prep.convert_delta_to_json.sql.connect')
+    @patch('scripts.data_prep.convert_delta_to_json.pd.DataFrame.to_json')
+    def test_stream_delta_to_json(self, mock_to_json: Any, mock_connect: Any):
```

Comment:
Does this need to be in a class? Most of our tests in foundry are not in classes, they're just top level functions